exercises/pangram/src/test/java/PangramsTest.java

```diff
@@ -8,62 +8,62 @@
 
 
     @Test
-    public void sentenceEmpty() {
+    public void emptySentenceIsNotPangram() {
         assertFalse(Pangrams.isPangram(""));
     }
 
     @Ignore
     @Test
-    public void pangramWithOnlyLowerCase() {
+    public void pangramWithOnlyLowerCaseLettersIsRecognizedAsPangram() {
         assertTrue(Pangrams.isPangram("the quick brown fox jumps over the lazy dog"));
     }
 
     @Ignore
     @Test
-    public void missingCharacterX() {
+    public void phraseMissingCharacterXIsNotPangram() {
         assertFalse(Pangrams.isPangram("a quick movement of the enemy will jeopardize five gunboats"));
     }
 
     @Ignore
     @Test
-    public void anotherMissingCharacterX() {
+    public void anotherPhraseMissingCharacterXIsNotPangram() {
         assertFalse(Pangrams.isPangram("the quick brown fish jumps over the lazy dog"));
     }
 
     @Ignore
     @Test
-    public void pangramWithUnderscores() {
+    public void pangramWithUnderscoresIsRecognizedAsPangram() {
         assertTrue(Pangrams.isPangram("\"the_quick_brown_fox_jumps_over_the_lazy_dog\""));
     }
 
     @Ignore
     @Test
-    public void pangramWithNumbers() {
+    public void pangramWithNumbersIsRecognizedAsPangram() {
         assertTrue(Pangrams.isPangram("\"the 1 quick brown fox jumps over the 2 lazy dogs\""));
     }
 
     @Ignore
     @Test
-    public void missingLettersReplacedByNumbers() {
+    public void missingLettersReplacedByNumbersIsNotPangram() {
```

Comment:
Sorry I didn't spot this the first time round but this would possibly be clearer if renamed to `phraseWithMissingLetters...` to make it consistent with the others?